src/app/shared/NoteCreateInline.svelte

```diff
@@ -82,25 +85,39 @@
 
     showPublishInfo(pubs[0])
     opts = {...defaultOpts}
-    compose.clear()
+
+    editor.commands.clearContent()
     saving = false
   }
+
+  onMount(() => {
+    const options = getEditorOptions({
+      submit: onSubmit,
+      element: editorElement,
+      submitOnEnter: true,
+      autofocus: true,
+    })
+
+    editor = new Editor(options)
+
+    editorLoading = editor.storage.fileUpload.loading
+  })
 </script>
 
 <form on:submit|preventDefault={() => onSubmit()}>
   <AltColor background class="z-feature flex gap-4 overflow-hidden rounded p-3 text-neutral-100">
     <PersonCircle class="h-10 w-10" pubkey={$pubkey} />
     <div class="w-full min-w-0">
-      <Compose placeholder="What's up?" bind:this={compose} {onSubmit} style="min-height: 3em;" />
-      <div class="p-2">
-        <NoteImages bind:this={images} bind:compose includeInContent />
-      </div>
+      <!-- placeholder="What's up?" -->
+      <Compose bind:element={editorElement} {editor} style="min-height: 3em;" />
       <div class="flex items-center justify-between">
         <div class="flex items-center justify-end gap-3">
           <i class="fa fa-cog cursor-pointer" on:click={() => options.setView("settings")} />
-          <ImageInput multi hostLimit={3} on:change={e => images.addImage(e.detail)}>
-            <i slot="button" class="fa fa-paperclip cursor-pointer" />
-          </ImageInput>
+          <button
+            class="hover:bg-white-l staatliches flex h-7 w-7 cursor-pointer items-center justify-center gap-2 whitespace-nowrap rounded bg-white px-6 text-xl text-black transition-all"
```

Comment:
This button should look like the others. Also, be careful about using `black` and `white`, which won't play well with light theme.